core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/io/PlaylistJsonTest.kt

```diff
@@ -220,8 +221,10 @@ class PlaylistJsonTest {
             minBy { snippet.publishedAt }.snippet into "earliest"
         }
 
-        res.columnsCount() shouldBe typed.columnsCount() + 1
-        res.getColumnIndex("earliest") shouldBe typed.getColumnIndex("items") + 1
+        withClue(res.columnNames() to typed.columnNames()) {
```

Comment:
wow what is this magic :o

Reply:
It's a kotest utility. When assertion fails, it'll also print a "clue". In this case pair of lists